Manual.md

```diff
@@ -781,6 +781,18 @@ with the character `r` in the `version` variable. The `distfiles`
 location will automatically be set as well as the package made to depend
 on `R`.
 
+- `gemspec` For packages that use
+[gemspec](https://guides.rubygems.org/specification-reference/) files for building a ruby
+gem and then installing it. The `gem_cmd` variable can be used to set the name of the gem
+binary, `configure_args` can be used to pass arguments during compilation. If your package
+does not make use of compiled extensions consider using the `gem` build style instead.
+
+- `gem` For packages that are installed using gems from [RubyGems](https://rubygems.org/).
+The `gem_cmd` variable can be used to set the name of the gem binary. `noarch` is set
+unconditionally and `distfiles` is set by the build style if the template does not so.
+If your gem provides extensions which must be compiled consider using the `gemspec`
+build style instead.
```

Comment:
> If your gem provides extensions 

Still not quite happy with this, please somehow make clear that these are extensions written in languages other than ruby and as such are arch specific.

Reply:
All extensions by ruby definition are arch specific.

Reply:
If it's written in Ruby then it is part of lib/ not ext/

Reply:
Ah, okay.